core/cli_helpers.py

```diff
@@ -628,7 +629,11 @@ def command_catalog(self):
                  FileCmd('sos_commands/openvswitch/'
                          'ovs-ofctl_-O_OpenFlow13_show_{bridge}'),
                  FileCmd('sos_commands/openvswitch/'
-                         'ovs-ofctl_show_{bridge}')],
```

Comment:
i think we need to keep this since it does exist in some sosreports, i have also never seen 10-13 in a sos but no harm in having it i guess.

Reply:
Updated.
Indeed it a is slim possibility for it to use versions prior of OF 1.3 but it is still used in some usecases.